docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
+----------------
+
+.. _collection-list:
+
+.. note:: This API requires :doc:`authentication <auth>`.
+
+This endpoint allows you to list all collections authored by this user.
+Access is limited to your own collections, or users with `Users:Edit` permission.
```

Comment:
Oh really? Is there a warning in the console though? It may be looking for a reference/identifier. If you search through the docs we use double ticks everywhere to render a `code` tag.